### DISCUSSION TITLE
Location/Path of the Certificate Store used by DBeaver

### DISCUSSION THREAD

**Discussion by himanshu-ram-punjabi** [dbeaver/dbeaver#19940] [Q&A] [answered]
Hi, Team

We are using a CASB Tool (Netskope) to monitor the traffic and I am not able to login to O365 in Dbeaver. If we disable the CASB tool then I am able to login. The solution we received through Netskope would be install the Netskope certificate in the Certificate Store from where the DBeaver would be validating the certificate so want to know the Location/Path of the Certificate Store used by DBeaver.

Hardware Overview:
Model Name: MacBook Pro
Model Identifier: MacBookPro17,1
Chip: Apple M1
Total Number of Cores: 8 (4 performance and 4 efficiency)
Memory: 16 GB
System Firmware Version: 8422.100.650
OS Loader Version: 7459.141.1

Thank you,
Himanshu Punjabi

**Comment by E1izabeth** [comment, marked as answer]
You can find a guide here https://dbeaver.com/docs/wiki/Importing-CA-Certificates-into-DBeaver/23.0/
